docs/usage/search.md

```diff
@@ -24,6 +24,31 @@ Also, using the `Advanced search` button, it is possible to directly put filters
 
 ![Advanced search](assets/advanced-search.png)
 
+### Extended search to uploaded files content
```

Comment:
```suggestion
### Extended search to upload files content
```

Reply:
uploaded files => files that are uploaded